### PR TITLE
SPCTR-3516: Fix - Icon List vertical layout issue in tablet

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

Icon List vertical layout breaks for tablet device.
https://brainstormforce.atlassian.net/browse/SPCTR-3516

### Screenshots

https://d.pr/v/4rw9u2

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,6 +215,9 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 ## Changelog ##
 
+### x.x.x ###
+* Fix: Icon List - Resolved an issue where the vertical layout alignment breaks in tablet device.
```
Comment (bsf-zanev) [inline README.md:219]:
```suggestion
* Fix: Icon List - Resolved an issue wherein the vertical layout alignment would not appear as intended in the editor's tablet view. 
```

File: src/blocks/icon-list/styling.js [hunk 2]
```diff
@@ -331,6 +331,7 @@ function styling( attributes, clientId, name, deviceType ) {
 		};
 		tabletSelectors[ ' .uagb-icon-list__wrap .block-editor-block-list__layout' ] = {
 			'flex-direction': 'column',
+			'align-items': 'flex-start',
```
Comment (bsf-zanev) [inline src/blocks/icon-list/styling.js:334]: It appears that this Mobile selector already has this - but does it need to be added for the Desktop selector of this type as well?

**Review by bsf-zanev — COMMENTED** [review]
Just a changelog update required, one doubt, and a conflict resolution.

File: src/blocks/icon-list/styling.js (hunk 2 above)
Comment (rajmohanreddy774) [reply src/blocks/icon-list/styling.js:334]: Not required, I have tested all the cases and it doesn't break for Desktop.

File: README.md (hunk 1 above)
Comment (rajmohanreddy774) [reply README.md]: Done.

**Review by bsf-zanev — APPROVED** [review]

**Review by AbhijitNage123 — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Pull request merged**